data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -185,7 +185,10 @@ def format_parties_data(data: dict) -> list[dict]:
             party_role['appointment_date'] = appointment_date
 
             cessation_date = None
-            if cessation_date := r['cp_cessation_dt_str']:
+            if role_code == 'LIQ':
+                if end_event_date := r['cp_end_event_dt_str']:
+                    cessation_date = end_event_date
```

Comment:
Lear omit time in party cessation_date, would it be ok to trim the time from end_event_date? what does the output says in colin?

Reply:
It's paper-only in colin and I only find one scanned image that shows date without time. I assumed lear would do something like what it does for [cease receiver](https://github.com/bcgov/lear/blob/main/queue_services/entity-filer/src/entity_filer/filing_processors/cease_receiver.py#L36) in the end so I keep the time. But yes I'll update the code to trim it to make it consistent.